### PR TITLE
Follow-up comments from Vincent

### PR DESCRIPTION
**Pull request by paulej**

### CODE REVIEW

File: private-media-framework/draft-ietf-perc-private-media-framework.md [hunk 1]
```diff
@@ -959,6 +971,11 @@ Thus what the media source said can be substantially delayed at the
 receiver with the receiver believing that it is what was said just
 now, and only delayed due to transport delay.
 
+While this attack cannot be eliminated entirely, its effectiveness
+can be reduced by re-keying the conference periodically since such
+significantly-delayed media would not be understood by endpoints that
+have expired old keys used to encrypt the media.
+
```
Comment (suhasHere) [inline private-media-framework/draft-ietf-perc-private-media-framework.md]: expired and old --> expired

File: private-media-framework/draft-ietf-perc-private-media-framework.md [hunk 2]
```diff
@@ -959,6 +971,11 @@ Thus what the media source said can be substantially delayed at the
 receiver with the receiver believing that it is what was said just
 now, and only delayed due to transport delay.
 
+While this attack cannot be eliminated entirely, its effectiveness
+can be reduced by re-keying the conference periodically since such
```
Comment (dabenham) [inline private-media-framework/draft-ietf-perc-private-media-framework.md]: I suggest shorten/reword this to " ... its effectiveness can be reduced by re-keying the conference periodically, since significantly-delayed media encrypted with expired keys would not be decrypted by endpoints.

File: private-media-framework/draft-ietf-perc-private-media-framework.md (hunk 1 above)
Comment (paulej) [reply private-media-framework/draft-ietf-perc-private-media-framework.md]: Thanks!  Done.

File: private-media-framework/draft-ietf-perc-private-media-framework.md (hunk 2 above)
Comment (paulej) [reply private-media-framework/draft-ietf-perc-private-media-framework.md]: Thanks! Done.
